search-website/README.md

```diff
@@ -19,44 +19,57 @@ To run the sample locally, follow the steps below.
+    - YOUR-SEARCH-RESOURCE-NAME
+    - YOUR-SEARCH-ADMIN-KEY - used in bulk import script
+    - YOUR-SEARCH-QUERY-KEY - used in Function api 
 - [Node.js and Git](https://nodejs.org/)
 - [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=shopathome-github-jopapa) installed
-- The [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions&WT.mc_id=shopathome-github-jopapa) installed
-- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=shopathome-github-jopapa) installed
+    - [Static Web app extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) installed
+    - [Cognitive Search extention](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecognitivesearch)
 
-### Setup
+### Download sample repository
 
-1. Clone (or Fork and Clone) this repository
-1. Open `./search-website` in Visual Studio Code. 
-1. Rename the `api/local.settings.json.rename` file to `api/local.settings.json`.
+1. In a terminal, use git to clone (or Fork and Clone) this repository to your local computer:
 
-    The `local.settings.json` file holds all of the keys that the application needs.
-    
-    For this sample, there is a search index using the [goodbooks-10k dataset](https://github.com/zygmuntz/goodbooks-10k) that that is publicly available using the connection information below. The index consists of 10,000 popular books that we'll search over in our application.
-    
-    ```json
-    {
-      "IsEncrypted": false,
-      "Values": {
-        "AzureWebJobsStorage": "",
-        "FUNCTIONS_WORKER_RUNTIME": "node",
-        "SearchApiKey": "REPLACE-WITH-YOUR-SEARCH-QUERY-KEY",
-        "SearchServiceName": "REPLACE-WITH-YOUR-SEARCH-RESOURCE-NAME",
-        "SearchIndexName": "good-books",
-        "SearchFacets": "authors*,language_code"
-      }
-    }
+    ```bash
+    git clone https://github.com/Azure-Samples/azure-search-javascript-samples
     ```
 
-## Run the app locally
+1. Open that local directory in Visual Studio Code.
 
-This project can be run anywhere, but VS Code is required for local debugging.
+### Run bulk import to the Search Index
 
-1. Open the `./search-website` application with VS Code.
+1. Open the `./search-website/bulk-insert/bulk_insert_books.js`.
+1. Download the [books.csv](https://raw.githubusercontent.com/zygmuntz/goodbooks-10k/master/books.csv) to the same directory as the `bulk_insert_books.js` file.
+1. Replace the following variables in the `bulk_insert_books.js` with your own values to authenticate with the Azure Search SDK:
 
-### Running the front-end
+    * YOUR-SEARCH-RESOURCE-NAME
+    * YOUR-SEARCH-ADMIN-KEY
 
-1. From a terminal in the `./search-website` root, install front-end dependencies...
+1. Open a terminal in VS Code from this directory by selecting the directory, then selecting **Open Integrated Terminal**.
```

Comment:
The option "Open in Integrated Terminal" is only available on folders, so I did this:  PS C:\Users\heidist\azure-search-javascript-samples\search-website\bulk-insert> npm install   

Obviously it's installing a boatload of software in that folder, which is kind of crazy. I have node_modules elsewhere, under %app-data%.   I'm going to blow this away and start over.

Reply:
So node-modules in app-data is globally installed and not local to the project. Most JS development is done local to the project and only CLIs or computer-wide tooling is installed globally. This project format of package.json in the exact project folder/subfolder is done on purpose and is common with JS repos that hold more than 1 project .